percona/pmm/pmm.go

```diff
@@ -94,6 +94,10 @@ func SidecarContainer(pgc *v2.PerconaPGCluster) corev1.Container {
 				Name:  "PMM_AGENT_SERVER_ADDRESS",
 				Value: pmmSpec.ServerHost,
 			},
+			{
```

Comment:
We need to add it only for crVer >= 2.7.0

Reply:
02742bd83a6f3c696318c5f68421a1e23057e996

And deleted dependence from the version for PMM_AGENT_PATHS_TEMPDIR because it was added more then 3 version before 2.7.0